deployment/aks-periscope.yaml

```diff
@@ -28,6 +28,12 @@ rules:
 - apiGroups: ["apiextensions.k8s.io"]
   resources: ["customresourcedefinitions"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["access.smi-spec.io", "specs.smi-spec.io", "split.smi-spec.io"]
+  resources: ["*"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["config.openservicemesh.io"]
+  resources: ["meshconfigs"]
+  verbs: ["get", "list", "watch"]
```

Comment:
@SanyaKochhar I seem to recall @arnaud-tincelin and @Tatsinnit mentioning that the `deployment/aks-periscope.yaml` file is being deprecated. I was previously told to make `aks-periscope-role ClusterRole` changes in https://github.com/Azure/aks-periscope/blob/master/deployment/cluster-role.yaml.

@Tatsinnit can you confirm?

Reply:
I did add these rbac permissions to `cluster-role.yaml` originally and I agree, it seems to make most sense to have them there, but it seemed like the rbac permissions doesn't work then. i.e., the `kubectl get meshconfig -A` command returns 
> Failed to collect meshconfigs for mesh osm: run command in container: exit status 1

Seeing similar error messaging for the SMI collector currently. Not sure if I'm missing some additional connection? @arnaud-tincelin @Tatsinnit please advise

Reply:
Discussed this with @Tatsinnit yesterday. I'll be moving the meshconfig ClusterRole to cluster-role.yaml so it can be deployed using Kustomize. Still testing this -- running into some issues with Kustomize currently

Reply:
Fixed this, moved cluster roles to cluster-role.yaml and tested this change with kustomize